### ISSUE TITLE
Dyskusje - strefy czasowe?

### ISSUE THREAD

**Issue by tomasz-grzesik** [allegro/allegro-api#8944]
Czy dziś jest tutaj coś zmieniane?
Na Allegro (przeglądarka) po dodaniu wiadomość miała podzinę +2. 
Niestety po API również nieoprawna godzina.

Proszę napisać co tam się dzieje i czy już po wszystkim.

**Comment by Lukasz-Zurek** [comment]
@tomasz-grzesik Tak, wczoraj przy okazji wdrażania zmian po naszej stronie, pojawiły się problemy z przetwarzaniem dat w różnych strefach czasowych. Problem nadal występuje. Dopytuję o status. Oczywiście wrócę do Ciebie z informacją.

W imieniu Serwisu przepraszam za zaistniałe utrudnienia.

**Comment by tomasz-grzesik** [comment]
Dziękuję za potwierdzenie. Wydaję się, że już wiadomości przechodzą poprawnie, ale mile widziana informacja o zakończeniu problemu.

**Comment by Lukasz-Zurek** [comment]
@tomasz-grzesik Tak, właśnie mam potwierdzenie, że około godziny temu weszła poprawka :)

**Comment by tomasz-grzesik** [comment]
Dziś mam dyskusję z rokiem 2050.

**Comment by Lukasz-Zurek** [comment]
@tomasz-grzesik Przekazałem już dalej i czekam na szczegóły. Dzięki za zgłoszenie!

**Comment by tomasz-grzesik** [comment]
last message at: 2050-01-31 23:00:00.967000+00:00
zarejestrowano: 2024-05-06 06:20:44.742877+00:00

**Comment by Lukasz-Zurek** [comment]
Właśnie otrzymałem potwierdzenie, że od pewnego czasu mogą się zdarzać niepoprawne zapisy (do kilku dziennie) i tu zespół odpowiedzialny za dyskusje analizuje przyczynę i jednocześnie stara się poprawiać te przypadki ręcznie.

Przepraszamy za te utrudnienia.

Jak już otrzymam oficjalne potwierdzenie wdrożenia globalnej poprawki dla tych błędów - poinformuję w wątku. Ale jednocześnie daj proszę znać, gdyby skala błędów nagle wzrosła.

**Comment by tomasz-grzesik** [comment]
Ok. Czekam zatem na poprawkę.
